redbot/core/help_formatter.py

```diff
@@ -20,25 +20,24 @@
 e.g. format_help_for(ctx, ctx.command, "Missing required arguments")
 
 discord.py 1.0.0a
-Experimental: compatibility with 0.16.8
 
 Copyrights to logic of code belong to Rapptz (Danny)
 Everything else credit to SirThane#1780"""
```

Comment:
These credits at the top don't seem accurate anymore. The majority of the logic below is not equivalent to the work Danny or SirThane did. I'd like to see this modified to more appropriately credit their work as the work this is now building off of, rather than all of this being their work.